src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -2038,6 +2055,7 @@ static int parse_args(vector<const char*>& args, std::ostream *err_msg,
   std::vector<const char*>::iterator i;
   std::ostringstream err;
   std::string arg_value;
+  int snapid;
```

Comment:
Let's not constrain the value to an int for no reason:
```suggestion
  long long snapid;
```